api/datastorage-api/src/main/java/com/cognifide/aet/vs/MetadataDAO.java

```diff
@@ -47,6 +47,14 @@
    */
   Suite getSuite(DBKey dbKey, String correlationId) throws StorageException;
 
+  /**
+   * @param dbKey - key with project and company name
+   * @param name - name of suite
+   * @param version Suite object found by given criteria or null.
```

Comment:
Please fix this javadoc - move `Suite object found by given criteria or null.` to `@return` and add description for `version` param